.travis.yml

```diff
@@ -0,0 +1,23 @@
+language: dart
+
+dart:
+  - dev
+  - stable
+
+dart_task:
+  #- test: --platform vm,chrome
+  - dartanalyzer: --fatal-infos --fatal-warnings .
```

Comment:
this task doesn't have a nice label in Travis. Maybe we don't care though?

Reply:
Known issue 😢